WalletWasabi.Fluent/ViewModels/CoinJoinProfiles/CoinJoinProfilesViewModel.cs

```diff
@@ -0,0 +1,69 @@
+using ReactiveUI;
+using System.Collections.Generic;
+using System.Reactive.Disposables;
+using System.Threading.Tasks;
+using System.Windows.Input;
+using WalletWasabi.Blockchain.Keys;
+using WalletWasabi.Fluent.ViewModels.AddWallet;
+using WalletWasabi.Fluent.ViewModels.Dialogs;
+using WalletWasabi.Fluent.ViewModels.Navigation;
+
+namespace WalletWasabi.Fluent.ViewModels.CoinJoinProfiles;
+
+[NavigationMetaData(Title = "CoinJoin Profiles")]
+public partial class CoinJoinProfilesViewModel : RoutableViewModel
+{
+	[AutoNotify] private CoinJoinProfileViewModelBase _selectedProfile;
+
+	public CoinJoinProfilesViewModel(KeyManager keyManager)
+	{
+		NextCommand = ReactiveCommand.Create(() => OnNext(keyManager));
+		EnableBack = true;
+
+		var privateProfile = new PrivateCoinJoinProfile();
+
+		Profiles = new()
+		{
+			privateProfile,
+			new SpeedyCoinJoinProfile(),
+			new EconomyCoinJoinProfile()
+		};
+
+		_selectedProfile = privateProfile;
+
+		ManualSetupCommand = ReactiveCommand.CreateFromTask(async () => await OnManualSetupAsync());
+	}
+
+	public ICommand ManualSetupCommand { get; }
+
+	public List<CoinJoinProfileViewModelBase> Profiles { get; }
+
+	private async Task OnManualSetupAsync()
+	{
+		var dialog = new ManualCoinJoinProfileDialogViewModel();
+
+		var dialogResult = await NavigateDialogAsync(dialog, NavigationTarget.CompactDialogScreen);
+
+		if (dialogResult.Result is { })
```

Comment:
@soosr is this a leftover?

Reply:
The dialog is not finished yet. It exists, but the content is empty.